site/SiteFooter.tsx

```diff
@@ -14,267 +14,173 @@ interface SiteFooterProps {
+                                height="62"
+                            />
+                        </a>
+                        <a href="https://www.ox.ac.uk">
+                            <img
+                                src={`${props.baseUrl}/oxford-logo.svg`}
+                                alt="University of Oxford logo"
+                                loading="lazy"
+                                width="58"
+                                height="62"
+                            />
+                        </a>
+                        <a href="https://global-change-data-lab.org">
+                            <img
+                                src={`${props.baseUrl}/gcdl-logo.svg`}
+                                alt="Global Change Data Lab logo"
+                                loading="lazy"
+                                width="106"
+                                height="62"
+                            />
+                        </a>
                     </div>
+                    <a href="https://www.ycombinator.com">
+                        <img
+                            src={`${props.baseUrl}/yc-logo.svg`}
+                            alt="Y Combinator logo"
+                            loading="lazy"
+                            width="123"
+                            height="30"
+                        />
+                    </a>
+                </div>
+            </div>
+            <div className="footer-right span-cols-6 span-sm-cols-12 col-start-8 col-sm-start-2 grid grid-cols-3 grid-sm-cols-2">
+                <div className="footer-link-column">
+                    <h5 className="h5-black-caps">Explore</h5>
+                    <FooterLinkList links={EXPLORE_LINKS} />
+                    <h5 className="h5-black-caps">Resources</h5>
+                    <FooterLinkList links={RESOURCE_LINKS} />
+                </div>
+                <div className="footer-link-column">
+                    <h5 className="h5-black-caps">About</h5>
+                    <FooterLinkList links={ABOUT_LINKS} />
+                </div>
+                <div className="footer-link-column">
+                    <h5 className="h5-black-caps">
+                        <FontAwesomeIcon icon={faRss} />
+                        RSS Feeds
+                    </h5>
+                    <FooterLinkList links={RSS_FEEDS} />
                 </div>
             </div>
+            <div className="footer-base span-cols-12 col-start-2 grid grid-cols-2 grid-sm-cols-1">
+                <div className="footer-base__socials">
+                    <h5 className="h5-black-caps">Follow us</h5>
+                    {SOCIALS.map((social) => (
+                        <a
+                            href={social.url}
+                            key={social.url}
+                            data-track-note="footer_social"
+                        >
+                            <FontAwesomeIcon icon={social.icon} />
```

Comment:
Ah, and also, slightly related:
do we want to [use `square-bluesky` for bluesky](https://fontawesome.com/icons/square-bluesky?f=brands&s=solid)? it's better sized than the oversized one we are using now, and fits better with the FB & LI icons next to it.

we might need to update fontawesome to get it, though.